sdk-java/src/test/java/com/spotify/confidence/ConfidenceIntegrationTest.java

```diff
@@ -0,0 +1,525 @@
+package com.spotify.confidence;
+
+import static com.spotify.confidence.ErrorType.*;
+import static com.spotify.confidence.ResolverClientTestUtils.generateSampleResponse;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.mock;
+
+import com.google.protobuf.util.Structs;
+import com.google.protobuf.util.Values;
+import com.spotify.confidence.ResolverClientTestUtils.ValueSchemaHolder;
+import com.spotify.confidence.shaded.flags.resolver.v1.FlagResolverServiceGrpc.FlagResolverServiceImplBase;
+import com.spotify.confidence.shaded.flags.resolver.v1.ResolveFlagsRequest;
+import com.spotify.confidence.shaded.flags.resolver.v1.ResolveFlagsResponse;
+import com.spotify.confidence.shaded.flags.resolver.v1.ResolvedFlag;
+import com.spotify.confidence.shaded.flags.types.v1.FlagSchema;
+import com.spotify.confidence.telemetry.FakeTelemetryClientInterceptor;
+import com.spotify.confidence.telemetry.Telemetry;
+import com.spotify.confidence.telemetry.TelemetryClientInterceptor;
+import com.spotify.telemetry.v1.LibraryTraces;
+import com.spotify.telemetry.v1.Monitoring;
+import io.grpc.ManagedChannel;
+import io.grpc.Metadata;
+import io.grpc.Server;
+import io.grpc.Status;
+import io.grpc.inprocess.InProcessChannelBuilder;
+import io.grpc.inprocess.InProcessServerBuilder;
+import io.grpc.stub.StreamObserver;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiConsumer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+final class ConfidenceIntegrationTest {
```

Comment:
We were missing good integration tests for resolving using the Confidence SDK standalone. This class was copied from the Provider tests and just massaged to fit the SDK APIs.